example/flask/templates/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bugsnag Python: Flask example</title>
+</head>
+
+<body>
+  <div>
+    <h1>Bugsnag Python: Flask example</h1>
+  </div>
+  <div>
+    <h3>This page is a basic example of how to include bugsnag-python in a flask app.
+    </h3>
+  </div>
+
+    <br>
+  <h3>Click below to send errors to your Bugsnag dashboard.</h3>
+  <br>
+  <br>
+    <div id="python">
+        <a href="/crashdict">
```

Comment:
We should have some styling on these `a` elements could be added to stop them from being underlined.
Something like `style="text-decoration:none"`.